### PR TITLE
Release note updates for Core 2.2.1

### PR DESCRIPTION
**Pull request by praveek**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/mobile-core/release-notes.md [hunk 1]
```diff
@@ -30,6 +48,10 @@ Please note that the current release includes the following changes:
 * Fixed in-app message redraw on device orientation change.
 * Fixed a memory leak with the in-app message webview.
 
+### Android Lifecycle 2.0.2
+
+* Chain Lifecycle events.
```
Comment (sbenedicadb) [inline src/pages/documentation/mobile-core/release-notes.md:53]: can we make this a little more descriptive?  what does it mean?  what is the purpose?

File: src/pages/documentation/release-notes/index.md [hunk 2]
```diff
@@ -244,6 +296,10 @@ Please note that the current release includes the following changes:
 * Fixed in-app message redraw on device orientation change.
 * Fixed a memory leak with the in-app message webview.
 
+### Android Lifecycle 2.0.2
+
+* Chain Lifecycle events.
```
Comment (sbenedicadb) [inline src/pages/documentation/release-notes/index.md:301]: same comment as before

**Review by calebk1 — COMMENTED** [review]
Couple of additional comments.

File: src/pages/documentation/mobile-core/release-notes.md [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+
+* Updated EventDataUtils to drop entries with unsupported values during cloning instead of throwing an exception.
+* Fixed a crash when displaying in-app messages.
+* Fixed rounded corners when displaying in-app messages.
+* Fixed an issue where the shared state published by Event Hub was mutable.
+
+### Android Lifecycle 2.0.3
+
+* Fixed an issue where session start time (`starttimestampmillis`) was shared in seconds instead of milliseconds in Lifecycle shared state.
+* Fixed an issue where session start time (`starttimestampmillis`), previous session start time, (`previoussessionstarttimestampmillis`) and previous session pause time (`previoussessionpausetimestampmillis`) were shared in seconds instead of milliseconds in Lifecycle response event.
```
Comment (calebk1) [inline src/pages/documentation/mobile-core/release-notes.md:15]: Oops, I told you to put the comma in the wrong place - the comma should be before the "and", not before the bracket. Sorry!

**Pull request merged**